docs/validators/cosmovisor.mdx

```diff
@@ -170,13 +166,13 @@ app.toml file. See the [CosmosSDK Telemetry
 Documentation](https://docs.cosmos.network/v0.46/core/telemetry.html) for more
 information.
 
-## ZetaVisor Environment Variables
+## Cosmovisor Environment Variables
 
-Breif explanation of the environment variables used by zetavisor.
+Breif explanation of the environment variables used by Cosmovisor.
 
 - `DAEMON_HOME`
 
-  - The location where the zetavisor/ directory is kept that contains the
+  - The location where the Cosmovisor/ directory is kept that contains the
     genesis binary, the upgrade binaries, and any additional auxiliary files
     associated with each binary (e.g. $HOME/.zetacored, etc.).
```

Comment:
Delete this whole section and replace it with 

## More Information About Cosmovisor 
For more detailed information about Cosmosvisor you can visit the [Cosmos Documentation Here](https://docs.cosmos.network/main/build/tooling/cosmovisor)